03-pipefilter.md

```diff
@@ -254,6 +254,8 @@ we get three processes with data flowing from the files,
 through `wc` to `sort`,
 and from `sort` through `head` to the screen.
 
+<img src="img/redirects-and-pipes.svg" alt="Redirects and Pipes" />
```

Comment:
For the .md files, we're using markdown image syntax: 
![Redirects and Pipes](fig/redirects-and-pipes.svg)